Create index.html

Vladislav7327 opened the pull request:
Написал HTML первой страницы сайта.

Vladislav7327 committed:
Create index.html

Написал HTML первой страницы сайта.

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <li><a href="#"><img src="search.png" alt="search-link.png"></a></li>
+                <li><a href="#">Contact  Us</a></li>
+            </ul>
+        </nav>
```
из nav нужно убрать img и второй список с поиском и contact-us. Они будут просто лежать в хедере и соседствовать с nav

Vladislav7327 replied:
Ок, есть

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+    </header>
+
+    <Main>
+        <section class="Collection">
```
не нужно названия классов писать с прописной буквы и не совсем понятно почему именно collection, это название ни оч чем не говорит, я бы назвал этот блок main-top потому что он на главной и в самом верху, и больше нигде не встречается и не может встречаться

Vladislav7327 replied:
ОК, не обратив внимание, что это слайдер - поправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+
+    <Main>
+        <section class="Collection">
+            <p class="Collection_somenumber">01</p>
```
можно "p" но я бы сделал лучше span 
и somenumber заменил бы index или counter

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <li><img src="line.png" alt="line"></li>
+                <li><img src="line.png" alt="line"></li>
+                <li><img src="line.png" alt="line"></li>
+            </ul>
```
этот блок называется pagination 
по идее можно не делать это списком, но вроде ничего страшного и можно оставить, но внутри li можно ничего не использовать, убирай картинку, палочку потом на css сделаешь

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <li><img src="line.png" alt="line"></li>
+                <li><img src="line.png" alt="line"></li>
+            </ul>
+            <h1 class="Collection_title">THE OOH NOO COLLECTION</h1>
```
это не h1, потому что этот блок слайдер, значит на каждом слайде верстка будет повторяться, а меняться будут только данные (заголовок, картинка, описание, возможно текст на кнопке). Это значит что h1 может быть только один на странице, а тогда он будет уже не один. Используй h2

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+            </ul>
+            <h1 class="Collection_title">THE OOH NOO COLLECTION</h1>
+            <p class="Collection_text">Designed in Slovenia</p>
+            <a class="Collection_button Button" href="#">EXPLORE NOW</a>  
```
нигде не пиши строчными буквами, все это потом сделаем стилями, везде просто обычный текст

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+            <h1 class="Collection_title">THE OOH NOO COLLECTION</h1>
+            <p class="Collection_text">Designed in Slovenia</p>
+            <a class="Collection_button Button" href="#">EXPLORE NOW</a>  
+            <!-- Уточнить кнопка или ссылка строка выше -->
```
все правильно ссылка, потому что ведет на другую старницу

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+            <h2 class="Category-Products_title">Category Products</h2>
+            <a class="Category-Products_link" href="#">
+                <div class="Category-Products_image-block block-1">
+                    <img class="Category-Products_image-block_image" src="shirt.png" alt="shirt">
```
я бы не ставил сюда эти картинки так как они не контентные а фоновые, значит мы просто можем добавить их стилями

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                </div>
+            </a>
+            <a class="Category-Products_link" href="#">
+                <div class="Category-Products_image-block block-2">
```
с классами пока тебе тяжело, поэтому предлагаю пока их не ставить лишних, а ставить потом по мере необходимости при стилизации, но еще раз скажу точно не нужны первые заглавные буквы и слишком сильно не пытайся удлинить названия.

Vladislav7327 replied:
Беру всегда какое-то название, либо заголовок блока, который оформляю. 
Логика такая - видишь контент на странице - берешь основной заголовок, или вроде того - и потом его добавляешь в код, чтобы можно было сразу понять где картинка сопоставляется с кодом.
Ты классы прописываешь местоположением блока на странице? 
Какие бы классы ты тут прописал?

ea-dorosh replied:
с классами прямо пока совсем плохо, следуй простым правилам, никаких заглавных букв, никаких длинющих названий типа Category-Products_image-block_image 
я бы написал так начиная с section class="products" не нужно "category-products"  потому что это простой загловок, завтра они напишут там category new year products ты же не будешь так класс называть
класс должен описывать сущность
section class="products"
    h2 class="products__title"
    ul class="products__list"
        li class="products__item"
           a class="products__item-link"
           h3 class="products__item-title"

ea-dorosh replied:
можешь попробовать мой пример перенести в свой код и увидишь что все станет более структурировано и тд, и сможешь дальше пытаться придумывать классы наподобие этих

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                </div>
+            </a>
+            <img class="Category-Products_left" src="left.png" alt="left">
+            <img class="Category-Products_right" src="right.png" alt="right">
```
это должен быть div class="navigation" а в нем две кнопки button опять же картинку сделать можно будет фоном то есть стилями

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <p class="Delivery_text-block-text">Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.</p>
+                <p class="Delivery_text-block-text">What started as an outlet to deal  with the traumatic loss of my husband, dlk has now become a truly happy place for me and i have you and all of the beautiful designers that we work with to thank!</p>
+            </article>
+            <button class="Delivery_text-block-button button">LEARN MORE</button>
```
это ссылка

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <li><img src="apple.png" alt="apple"></li>
+                <li><img src="eltex.png" alt="eltex"></li>
+                <li><img src="owox.png" alt="owox"></li>
+            </ol>
```
ol?? ol это нумерованный список, ul - ненумерованный, здесь нет нумерации значит ul

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+        </section>
+
+        <section class="About-us">
+            <article class="About-us_Design-Life-Kids">
```
article неверный тэг в данном месте по идее это section и CONTACTS тоже section а лежат они вместе в одном div

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <h2 class="About-us_Design-Life-Kids_title">Design Life Kids</h2>
+                <p class="About-us_Design-Life-Kids_text italic">About Us</p>
+                <p class="About-us_Design-Life-Kids_text">Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.</p>
+                <button class="About-us_Design-Life-Kids_button button">buy now</button>
```
ссылка

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                    Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.
+                </p>
+            </article>
+            <img class="About-us_map" src="map.png" alt="map">
```
почему карта у тебя выпала из блока?

Vladislav7327 replied:
В фигме висит отдельным куском каки-то, вот и сунул отдельно от основных блоков.
Ублрал внутрь

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,173 @@
+                <li><a href="#">Web Design — Direct Line Development</a></li>
+            </ul>
+        </div>
+    </footer>
```
футер совсем не получился, надо все убрать и заново попробовать, ты разделил его на верх середину и низ, а надо было на колонки. Я попытался разделить их визуально, посмотри, понятно ли http://joxi.ru/nAyZNDxCjzXper
плюс убирай все img (line, star  и тд) это все сделаем стилями

Vladislav7327 replied:
Нуу, вроде как задумку я понял.
Поправил - глянь, пожалуйста.

ea-dorosh replied:
да, вроде нормально, потом увидим лучше когда начнешь строить сетку, возможно что-то добавим или уберем

Vladislav7327 committed:
Update index.html

Добавил изменения по оставленным комментам.

Не трогал только заглавные буквы в названиях классов и длинные названия классов.  Буду это править по мере добавления стилей

ea-dorosh commented on index.html:
```diff
@@ -87,87 +81,78 @@ <h2 class="Delivery_title">Delivery</h2>
+                <li><button class="star"></button></li>
+                <li><button class="star"></button></li>
+                <li><button class="star"></button></li>
+            <!-- кнопка же это будет? 5 шт звезд? -->
```
нет, просто li пустые все остально потом стилями

Vladislav7327 replied:
+

ea-dorosh commented on index.html:
```diff
@@ -7,67 +7,61 @@
             </a>
-            <img class="Category-Products_left" src="left.png" alt="left">
-            <img class="Category-Products_right" src="right.png" alt="right">
+            <div class="Category-Products_navigation"></div>
```
ну а теперь в этот navigation нужно положить две кнопки переключения

Vladislav7327 replied:
+

Vladislav7327 committed:
Новые изменения

Подправил классы первой страницы, залил контент-картинки, разметил вторую страницу html.
